MetamodelAlgemeen.md

```diff
@@ -198,16 +199,47 @@ Diagram: Keuze tussen attribuutsoorten binnen een attribuutsoort. Patroon geldt
 
 *Use case 4: Keuze tussen relatiedoelen, als nadere invulling van 1 betekenisvolle relatiesoort*
 
-Dit diagram is uitgewerkt voor Objecttype. Voor Gegevensgroeptype geldt hetzelfde patroon, behalve dat een Gegevensgroeptype geen doel mag zijn van een Relatiesoort.
+Deze use case komt in de praktijk weinig voor, maar het kan wel, daarom is deze opgenomen. 
 
-Een objecttype of gegevensgroep kan normaal een relatiesoort hebben, die gekoppeld is aan een objecttype. Als er sprake is van een relatiesoort met een keuze, dan is het relatiedoel van de relatiesoort niet gekoppeld aan 1 objecttype, maar dan is het objecttype gekoppeld aan een keuze en deze keuze geeft 2 of meer relatiedoelen aan.
+Het bron object heeft 1 relatie, en heeft 1 gerelateerd doel object, maar dit doel object kan van verschillende objecttypes zijn. Het bron objecttype kent slechts 1 relatiesoort/relatierol, met een eigen betekenis, en deze relatiesoort/rol verwijst naar 2 of meer objecttypes. 
+
+Dit kan voorkomen wanneer vanuit het bron objecttype gezien er maar sprake is van 1 concern dat ingevuld moet worden, ongeacht welk objecttype dit concern zal gaan invullen. 
```

Comment:
Met het  woord concern introduceer je een nieuwe term. Dat is niet handig. Voorstel vervangen door "kenmerk":
Dit kan voorkomen wanneer vanuit het bron objecttype gezien er maar sprake is van 1 kenmerk dat ingevuld moet worden, ongeacht welk objecttype dit kenmerk zal gaan invullen.